projects/packages/calypsoify/src/scss/style-gutenberg.scss

```diff
@@ -5,7 +5,8 @@
  * Hide the last item of the first group when the screen is big enough.
  */
 @media (min-width: 782px) {
-  .components-dropdown-menu__menu:first-child .components-menu-group:first-child .components-button:last-of-type {
+  .interface-more-menu-dropdown__content .components-dropdown-menu__menu:first-child .components-menu-group:first-child .components-button:last-of-type,
+  .more-menu-dropdown__content .components-dropdown-menu__menu:first-child .components-menu-group:first-child .components-button:last-of-type {
```

Comment:
This prevents an issue with other dropdown menu items being hidden when Calypsoified (for example 'Mobile' when clicking the Preview icon). See D149786-code for more context.
Two different selectors are used because `.interface-more-menu-dropdown__content` is the correct classname in WordPress core (also used in Jetpack), but `.more-menu-dropdown__content` is what is used on Atomic and Simple. Keeping the core version for additional safety.